### DISCUSSION TITLE
Ereignisanzeige, Event ID 223

### DISCUSSION THREAD

**Discussion by Josef1966** [hoylabs/OpenDTU-OnBattery#1380] [Q&A]
Woher kommen diese Meldungen in der OpenDTU? WR = HMS-1600-4T, angeschlossen an die 4 Eingänge ist ein Zendure Hub 2000, dynamische Regelung über WR.
![IMG_1933](https://github.com/user-attachments/assets/4858c77c-be48-46df-ad15-5b142c9a0998)

**Comment by Tomv888** [comment]
Würde mich auch interessieren.Hast du auch die neue Firmware 1.1.12 drauf?
Seitdem kommt das nämlich bei mir durchschnittlich alle 2 Tage zur Morgendämmerung.
Denke das hat was mit Unterspannung an den PV Eingängen zu tun.

**Reply by Josef1966** [reply]
OpenDTU-Firmware = 2024.10.22
HMS-Firmware = 01.01.12

Die Vermutung habe ich auch, aber bestätigen konnte mir das bisher keiner.
Wie hast du dein System "verkabelt"

**Comment by Tomv888** [comment]
4x 570 Watt am HMS2000 und separater AC Anschluß.
Die Fehlerbeschreibungen in den HMS Bedienungsanleitungen gehen aktuell nur bis ID 222 oder so.
Da die FW 1.1.12 recht neu ist,taucht ID223 noch nirgends auf.Da müssen wir uns wohl noch gedulden

**Comment by schlimmchen** [comment]
Schaut bitte mal im [Upstream Projekt](https://github.com/tbnobody/OpenDTU), ob das dort schon gemeldet/besprochen wurde. Und wenn nicht, dann wird das dort noch behandelt werden und die Meldung wird dann auch in OpenDTU-OnBattery eine textuelle Beschreibung haben.
